src/main/java/ch/njol/skript/expressions/ExprAlphabetList.java

```diff
@@ -41,7 +37,7 @@
 
 @Name("Alphabetical Sort")
 @Description("Sorts given strings in alphabetical order.")
-@Examples({"set {_list::*} to alphabetically sorted {_list::*"})
+@Examples({"set {_list::*} to alphabetically sorted {_players::*}"})
```

Comment:
Note that this is the old variant of sort expression. It cannot process anything but strings.